habitat/plan.sh

```diff
@@ -1,5 +1,5 @@
 pkg_name=chef-workstation
-pkg_origin=chef
+pkg_origin=ngupta26
```

Comment:
Would this need to change based on where the package is released after a ci pipeline is in place?

Reply:
yes we will be updating all origin name from ngupta26/ to chef/ origin once the pipeline are in place.